src/components/views/dialogs/AccessTokenDialog.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2017 Vector Creations Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { _t } from '../../../languageHandler';
+import QuestionDialog from './QuestionDialog';
+
+type IProps = Exclude<
+    React.ComponentProps<QuestionDialog>,
+    "title" | "danger" | "description"
+    >;
```

Comment:
this line-breaking looks a bit strange

Reply:
You alright with https://github.com/matrix-org/matrix-react-sdk/pull/5355/commits/a3212c0477ec0fc9166a56a49a9579bae4712d5f?